src/services/MixService.php

```diff
@@ -80,6 +91,10 @@ public function init()
      */
     public function version($file)
     {
+        if (file_exists($this->assetFullPath.'/hot')) {
```

Comment:
`file_exists($this->assetFullPath.'/hot')` => `file_exists($this->assetFullPath . '/hot')`